app/views/admin/dashboard/show.html.erb

```diff
@@ -12,12 +12,13 @@
   </tr>
   <% @facade.tutorials.each do |tutorial| %>
     <tr class="admin-tutorial-card">
-      <td><%= link_to tutorial.title, tutorial_path(tutorial) %></td>
+      <td><%= link_to tutorial.title, tutorial_path(tutorial) %><br><br><hr></td>
```

Comment:
```suggestion
      <td><%= link_to tutorial.title, tutorial_path(tutorial) %></td>
```
I would rather do this styling with css. What do you think?

Reply:
I totally agree, this was a bit of a quick fix. I'll commit the suggestion!